api/services/registration_service.py

```diff
@@ -0,0 +1,43 @@
+# This file is treated as service layer
+import requests
+from flask import request
+from api import models, db
+
+
+class RegistrationService:
+    # Create a registration
+    def create_registration(self, data):
+
+        url_decoded_system_phone = requests.utils.unquote(data["to_number"])
+        system_phone_exists = db.session.query(
+            db.exists().where(models.SystemPhone.phone == url_decoded_system_phone)
+        ).scalar()
+
+        if not system_phone_exists:
+            return
+
+        system_phone_data = models.SystemPhone.query.filter_by(

```

Comment:
Do we need to get these details again? are we not getting these details in line 12?